app/js/DeploymentManager.js

```diff
@@ -0,0 +1,59 @@
+const logger = require('logger-sharelatex')
+const settings = require('settings-sharelatex')
+const fs = require('fs')
+
+// Monitor a status file (e.g. /etc/real_time_status) periodically and close the
+// service if the file contents don't contain the matching deployment colour.
+
+const FILE_CHECK_INTERVAL = 5000
+const statusFile = settings.deploymentFile
+const deploymentColour = settings.deploymentColour
+
+var serviceCloseTime
+
+function updateDeploymentStatus(fileContent) {
+  const closed = fileContent && fileContent.indexOf(deploymentColour) === -1
```

Comment:
I think we can do `fileContent.includes(deploymentColour)` now

Reply:
Ah, useful.  I will change it.